plotRisk.py

```diff
@@ -74,4 +80,4 @@ def heal(): # searches food items for needed nutritional value then consumes the
 char.magicDefence = 100
 char.defence = 100
 char.health = 10
```

Comment:
Is there a reason you don't put these in the constructor?

Reply:
Because our character should have self defined stats. Like mage would have higher magic attack than a barb. I "Leveled up" the character so that it would show that the attacks now have a lower risk